src/js/controller/projectCtrl.js

```diff
@@ -0,0 +1,63 @@
+let projects = [];
+
+function updateProjects() {
+  const localStorageArr = JSON.parse(localStorage.getItem('projects'));
+  if (localStorageArr) {
+    projects = [...localStorageArr];
+  }
+}
+
+function generateID(array) {
+  let ID;
+  if (array.length > 0) {
+    ID = array[array.length - 1].id + 1;
+  } else if (array.length === 0) {
+    ID = 0;
+  }
+  return ID;
+}
+
+function getProjectArr() {
+  return projects;
+}
+
+function addProject(newProject) {
+  projects.push(newProject);
+}
+
+function deleteProject(id) {
+  const ids = projects.map(current => current.id);
+
+  const index = ids.indexOf(id);
+  if (index !== -1) {
+    projects.splice(index, 1);
+  }
+}
+
+function editProject(index, title) {
+  projects[index].title = title;
+}
+
+function updateLocalStorage() {
+  localStorage.setItem('projects', JSON.stringify(projects));
+}
+
+function findProject(id) {
+  updateProjects();
+  const ids = projects.map(current => current.id);
+  const index = ids.indexOf(parseInt(id, 10));
+  return projects[index];
+}
+
+function validateInput(element) {
+  if (element.value === '') {
+    // element.classList.add('alert');
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

Reply:
Changes made! Thanks for the review!